p4_symbolic/symbolic/util.cc

```diff
@@ -172,113 +173,6 @@ gutil::StatusOr<SymbolicTrace> MergeTracesOnCondition(
   return merged;
 }
 
```

Comment:
This code is not really removed, it is moved to another file (values.cc) for organization purposes. I mark its location there with a comment to simplify review.